src/routes/Send.tsx

```diff
@@ -530,6 +534,16 @@ export default function Send() {
                         tags
                     );
 
+                    sentDetails.amount = amountSats();
+                    sentDetails.destination = address();
+                    sentDetails.txid = txid;
+                    sentDetails.fee_estimate = feeEstimate() ?? 0;
+                } else if (payjoinEnabled()) {
+                    const txid = await state.mutiny_wallet?.send_payjoin(
```

Comment:
should we add a try-catch block to enable that fallback to normal send? 
```
try {
const txid = await state.mutiny_wallet?.send_payjoin(
                        destination()!.original,
                        amountSats(),
                        tags
                    );
} catch {
const txid = await state.mutiny_wallet?.send_to_address(
                        address()!,
                        amountSats(),
                        tags
);
}
```
though this brings up the comment you made in the mutiny-node about where the waila should parse the receive string as some parsing would be needed between `destination.original()` and `address()`

Reply:
easy enough to just regex away the unimportant stuff, but still worth considering

Reply:
I'm not sure we should assume the user wants to spend as naive tx if they can avoid it. some automated payment processors will automatically spend the naive tx if payjoin fails but senders may want to try another utxo or choose something else like ln instead of immediately sending w/o a privacy preserving measure

Reply:
right, that makes sense